blog/2024/terraform-provider-adoption.adoc

```diff
@@ -0,0 +1,40 @@
+:title: Keycloak Adopts Terraform Provider
+:date: 2024-12-02
+:publish: false
+:author: Thomas Darimont
+
+# New Repository
+
+We're excited to announce that the Keycloak Terraform Provider has officially moved under the Keycloak organization! 
+You can find the new repository location https://github.com/keycloak/terraform-provider-keycloak[here].
+
+# The Journey So Far
+
+Thanks to https://www.keycloak.org/2024/09/realm-config-management-tools-survey-results.html[our community survey], we confirmed that the Keycloak Terraform Provider by mrparkers is the most widely used tool 
```

Comment:
```suggestion
Thanks to https://www.keycloak.org/2024/09/realm-config-management-tools-survey-results.html[our community survey], we confirmed that the Keycloak Terraform Provider by [mrparkers](https://github.com/mrparkers) is the most widely used tool 
```

Reply:
+1 for linking the previous maintainer. Please double-check the syntax, IMHO it is using a Markdown syntax here where it should use an AsciiDoc syntax.

Reply:
@ahus1 Thanks for pointing out the syntax! Yeah, I believe I used an incorrect one (Markdown).

@thomasdarimont Can you please if my mistake? :)

Reply:
I already fixed the links / syntax :)